bigquery_etl/docs/generate_docs.py

```diff
@@ -74,9 +75,13 @@ def main():
     """Generate documentation for project."""
     args = parser.parse_args()
     out_dir = os.path.join(args.output_dir, "docs")
-    if os.path.exists(out_dir):
+    override_dir = os.path.join(args.output_dir, "overrides")
```

Comment:
What is the purpose of the overrides directory?

Reply:
It's so that we can override the default home page. I made some customization to the main page so an override was needed. I will add a comment here to explain it.